.github/dependabot.yml

```diff
@@ -3,4 +3,9 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: daily
+    interval: monthly
+  ignore:
+    - dependency-name: "@types/node"
```

Comment:
I'm not sure but we should not update @types/node because we are now specifying Node.js 14+ in package.json.